src/test/java/org/folio/services/kafka/topic/KafkaAdminClientServiceTest.java

```diff
@@ -43,7 +43,7 @@ public void shouldNotCreateTopicIfAlreadyExist(TestContext testContext) {
     createKafkaTopicsAsync(mockClient)
       .onFailure(testContext::fail)
       .onComplete(testContext.asyncAssertSuccess(notUsed -> {
-        verify(mockClient, times(0)).createTopics(anyList());
+        verify(mockClient, times(1)).createTopics(anyList());
```

Comment:
Given that the topics are now _meant to be created_ I think the name should be changed to reflect that intent

Reply:
yep